src/store/transforms/transforms.ts

```diff
@@ -99,3 +103,21 @@ export const transformCircular = createTransform(
   inboundState => Flatted.stringify(inboundState),
   outboundState => Flatted.parse(outboundState),
 );
+
+export const transformContacts = createTransform(
+  inboundState => inboundState,
+  (_outboundState, k) => {
+    if (k === 'list') {
+      const contacts = _outboundState as ContactRowProps[];
+      const migratedContacts = contacts.map(contact => ({
+        ...contact,
+        chain:
+          contact.chain || BitpaySupportedUtxoCoins[contact.coin]
```

Comment:
This requires a parenthesis around `(BitpaySupportedUtxoCoins[contact.coin] ? contact.coin : 'eth')`, otherwise it will always resolve to `contact.coin` even when there is a `chain` set

Reply:
Sorry about that. Thanks